app/Policies/RegistrationPolicy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Policies;
+
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class RegistrationPolicy
+{
+    use HandlesAuthorization;
+
+    // These are permissions that we need to check a user for.
```

Comment:
Will they ever be different?

Reply:
We were specifically asked to treat them differently. I suspect ARC has some kind of future purpose for that.